examples/petsc/src/libceedsetup.c

```diff
@@ -48,25 +48,19 @@ PetscErrorCode SetupLibceedByDegree(DM dm, Ceed ceed, CeedInt degree,
   CeedQFunction qf_setup_geo, qf_apply;
   CeedOperator op_setup_geo, op_apply;
   CeedVector x_coord, q_data, x_ceed, y_ceed;
-  CeedInt P, Q, num_qpts, c_start, c_end, num_elem,
+  CeedInt num_qpts, c_start, c_end, num_elem,
           q_data_size = bp_data.q_data_size;
   CeedScalar R = 1,                      // radius of the sphere
              l = 1.0/PetscSqrtReal(3.0); // half edge of the inscribed cube
 
+  ierr = DMSetCoordinateDim(dm, topo_dim); CHKERRQ(ierr);
```

Comment:
That name does not seem right.

Reply:
https://petsc.org/release/docs/manualpages/DM/DMSetCoordinateDim.html

We found this function and added this line years ago, but I don't recall why

Reply:
Can you try commenting this out?

Reply:
I did comment it out and works fine since `topo_dim` is the input variable of `SetupLibceedByDegree` function and we set it by `rp->dim` in bps.c. I'll delete it.